dss/util/dynamodb.py

```diff
@@ -0,0 +1,63 @@
+from dss.util.aws.clients import dynamodb  # type: ignore
```

Comment:
Let's move this file to `dss/dynamodb/__init__.py`

Reply:
Alright.